game/scripts/npc/items/item_azazel_wall_4.txt

```diff
@@ -0,0 +1,67 @@
+"DOTAItems"
+{
+    //=================================================================================================================
+    // Recipe: Wall 4
+    //=================================================================================================================
+    "item_recipe_azazel_wall_4"
+    {
+        "ID"                            "8509"
+        // General
+        //-------------------------------------------------------------------------------------------------------------
+        "BaseClass"                     "item_datadriven"
+        "Model"                         "models/props_gameplay/recipe.vmdl"
+        "ItemCost"                      "100"
+        "ItemRecipe"                    "1"
+        "ItemResult"                    "item_azazel_wall_4"
+        "AbilityTextureName"            "custom/recipe/recipe_4"
+        "ItemRequirements"
+        {
+            "01"                        "item_upgrade_core_4"
+        }
+    }
+    //=================================================================================================================
+    // Azazel: Wall 4
+    //=================================================================================================================
+    "item_azazel_wall_4"
+    {
+        "ID"                            "8497"
+        // General
+        //-------------------------------------------------------------------------------------------------------------
+        "BaseClass"                     "item_lua"
+        "ScriptFile"                    "items/azazel_wall.lua"
+        "AbilityTextureName"            "custom/spirit_vessel_4"
+        "AbilityBehavior"               "DOTA_ABILITY_BEHAVIOR_POINT"
+        "AbilityCastRange"              "600"
+        "ItemPermanent"                 "0"
+        "ItemInitialCharges"            "1"
+        "MaxUpgradeLevel"               "4"
+        "ItemBaseLevel"                 "4"
+        "ItemStackable"                 "1"
+        "ItemCost"                      "100"
+        // Special
+        //-------------------------------------------------------------------------------------------------------------
+        "AbilitySpecial"
+        {
+            "01"
+            {
+                "var_type"                  "FIELD_INTEGER"
+                "construction_time"         "10"
+            }
+            "02"
+            {
+                "var_type"                  "FIELD_INTEGER"
+                "bonus_health"              "0 4500 10500 22500"
```

Comment:
You're already using `SetMaxHealth`. I think the KV might as well be just the total health.

Reply:
Yeah, I'll do this.